dev/com.ibm.ws.componenttest/autoFVT-defaults/src/ant/launch.xml

```diff
@@ -102,8 +102,8 @@
 		<!-- In Java 16, the default enforcement of strong encapsulation (part of JPMS, which was introduced in Java 9) has changed from 
 		     illegal-access=permit to illegal-access=deny.  This causes many of our FATs that use reflection to fail.  
 		     This change sets up a gated property (illegal.access.permit) so that any builds that take place on Java 15+ use 
-		     illegal-access=permit, but older versions will be skipped -->
-		<condition property="illegal.access.permit" value="--illegal-access=permit">
+		     illegal-access=permit, but older versions will be skipped --> 
+		<condition property="illegal.access.permit" value="--add-opens java.base/java.lang=ALL-UNNAMED">
```

Comment:
So we are still going to globally set `--add-opens java.base/java.lang=ALL-UNNAMED` for all FATs instead of just the FATs where it is needed?

Reply:
Yes, as we have over 350 FATs that need this jvm override to be able to run at Java 16+.